spec/integration/filtering_spec.rb

```diff
@@ -66,6 +66,47 @@ def run_rerun_command_for_failing_spec
   end
 
   context "passing a line-number filter" do
+    it "works with different custom runners used in the same process" do
+      class Foo
+        attr_accessor :i
+
+        def initialize(*)
+          @i = 0
+        end
+
+        def example_passed(notification)
+          @i += 1
+        end
+      end
+
+      write_file_formatted 'spec/filtering_custom_runner_spec.rb', "
+        RSpec.describe 'A group' do
+          example('ex 1') { }
+          example('ex 2') { }
+        end
+      "
+
+      RSpec::Core::Formatters.register(Foo, :example_passed)
+
+      formatter = Foo.new
+      RSpec.configuration.add_formatter(formatter)
+      opts = RSpec::Core::ConfigurationOptions.new(
+        ["tmp/aruba/spec/filtering_custom_runner_spec.rb[1:1]"])
```

Comment:
@JonRowe I suspect there's a better way of doing this instead of hard-coding `tmp/aruba/` but I couldn't find it. Any ideas?

Reply:
Aruba has a DSL, it includes `expand_path('file')` which returns `# => <path>/tmp/aruba/file`, that would seem like an easy win here.
https://www.rubydoc.info/gems/aruba/Aruba%2FApi%2FCore:expand_path